README.md

```diff
@@ -24,33 +24,33 @@ cd mpact-compiler
 git submodule update --init --recursive --progress
 ```
 
-All following commands assume you remain in the `mpact-compiler` directory.
+NOTE: All of the following commands assume you remain in the `mpact-compiler` directory.
 
 ### Setup Python virtual environment
 
-The following commands initialize a virtual environment.
+The following commands initialize a virtual environment under bash/sh/etc.  For other shells, see Note 1, [below](README.md#notes).
 
 ```shell
 python3.11 -m venv mpact_venv  # one time set up
-source mpact_venv/bin/activate # for each session
+source mpact_venv/bin/activate  # MUST BE REPEATED FOR EVERY SESSION
 ```
 
-Also make sure to set the Python paths as follows.
-
+Next, set the Python paths as follows; for shells not in the bash/sh family, see Note 2, [below](README.md#notes).
 ```shell
-export PYTHONPATH=`pwd`/build/tools/mpact/python_packages/mpact
+export PYTHONPATH=`pwd`/build/tools/torch-mlir/python_packages/torch_mlir:pwd`/build/tools/mpact/python_packages/mpact
```

Comment:
Is the first path needed? It seems like that I could run with just the second path alone.

Reply:
yeah, I believe

export PYTHONPATH=`pwd`/build/tools/mpact/python_packages/mpact

should just work

Reply:
Tim, can you elaborate why this as added?